### DISCUSSION TITLE
Cancel store event

### DISCUSSION THREAD

**Discussion by AitorMillan** [GrapesJS/grapesjs#5931] [Q&A]
Is there a way to cancel the GrapesJS store event? I want to capture or be able to cancel the event and run my own logic.

Thanks in advance!

**Comment by artf** [comment]
Did you try the [onStore](https://github.com/GrapesJS/grapesjs/blob/679e23cb2508d836e28be181153dadd37c706065/src/storage_manager/config/config.ts#L63) config?
